saleor/dashboard/order/views.py

```diff
@@ -6,13 +6,13 @@
 from django.shortcuts import get_object_or_404, redirect
 from django.template.context_processors import csrf
 from django.template.response import TemplateResponse
-from django.urls import reverse
 from django.utils.translation import pgettext_lazy
 from django_prices.templatetags.prices_i18n import gross
 from payments import PaymentStatus
 from prices import Price
 from satchless.item import InsufficientStock
 
+from saleor.product.models import StockLocation
```

Comment:
This should be a relative import as we don't use absolutes outside of our tests suite.

Reply:
Yes, you're right. I must have missed it.